test/performance/odp_l2fwd_run.sh

```diff
@@ -76,7 +76,7 @@ run_l2fwd()
 	LOG=odp_l2fwd_tmp.log
 
 	# Max 2 workers
-	odp_l2fwd${EXEEXT} -i $IF1,$IF2 -m 0 -t 1 -c 2 | tee $LOG
+	odp_l2fwd${EXEEXT} -i $IF1,$IF2 -m 0 -t 1 -c 2 -w 10 | tee $LOG
```

Comment:
With `l2fwd`, the wait shortopt was `-S`, though this issue disappears with longopts only.